renovate.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":timezone(Asia/Tokyo)",
+    ":automergeMinor",
+    ":automergeRequireAllStatusChecks"
+  ],
+  "prConcurrentLimit": 5,
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["^latex-utensils$"],
+      "automerge": true,
+      "minor": {
+        "automerge": false
+      }
+    }
```

Comment:
The rule describes that minor updates will be merged automatically, excepting latex-utensils and of which the bot merges patch level updates automatically, right?